x-pack/libbeat/management/manager.go

```diff
@@ -112,29 +112,36 @@ func (cm *ConfigManager) worker() {
 	defer cm.wg.Done()
 
 	// Initial fetch && apply (even if errors happen while fetching)
-	cm.fetch()
-	cm.apply()
```

Comment:
I think this looked easier to understand to me, it also didn't need the additional handling of `firstRun` in the loop. Is there any reason to change it?

Reply:
I realized I also have to handle a store if fetch brings new configs, which complicated the overall logic. Having the 3 steps repeated in the code looks like a bad idea

Reply:
Oh ok, so the difference is that now it can also trigger a Save on first run.

Reply:
yep, I would say it was a bug before